CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 * [#1216](https://github.com/ruby-grape/grape/pull/1142): Fix JSON error response when calling `error!` with non-Strings - [@jrforrest](https://github.com/jrforrest).
 * [#1225](https://github.com/ruby-grape/grape/pull/1225): Fix `given` with nested params not returning correct declared params - [@JanStevens](https://github.com/JanStevens).
 * [#1249](https://github.com/ruby-grape/grape/pull/1249): Don't fail even if invalid type value is passed to default validator - [@namusyaka](https://github.com/namusyaka).
+* [#1263](https://github.com/ruby-grape/grape/pull/1263): Refactor generated OPTIONS and Not Allowed routes to fix 404 handling - [@arempe93](https://github.com/arempe93)
```

Comment:
I think the important part is that this fixes handling of 404 with OPTIONS, so I would just say that, eg. "Fix: using route :any breaks OPTIONS" or something like that.